### PR TITLE
Add a button to move education fields

### PR DESCRIPTION
**Pull request by gabivlj**
I think it would be cool to have a quick button to move up education fields so you can quickly swap educations. It would also be cool to translate this to other forms. Here is a screenshot of the implementation:
image
image
After clicking the up button it moves up. 

I think the CSS and design could get so much better ofc. If this is something that is interesting we could improve how it looks.

### CODE REVIEW

**Review by ducaale — COMMENTED** [review]
>I think the CSS and design could get so much better ofc. If this is something that is interesting we could improve how it looks

We are currently porting Resumake's old design to v3 so no need to worry about the styling

File: src/components/generator/form/sections/EducationSection.tsx [hunk 1]
```diff
@@ -3,16 +3,31 @@ import { FormSection } from './FormSection'
...
+  )
+
   return (
     <FormSection title="Your Educational Background">
       {fields.map((field, index) => (
-        <Card key={field.id} removeCard={() => remove(index)}>
+        <Card
+          key={field.id}
+          removeCard={() => remove(index)}
+          moveUp={index >= 1 ? () => upFunction(index) : undefined}
```
[the remark this answers is not in the document]
Comment (gabivlj) [reply src/components/generator/form/sections/EducationSection.tsx:29]: I think re-rendering the entire form is not good when we swap positions, I'm wondering why is that happening.

[the remark this answers is not in the document]
Comment (gabivlj) [reply src/components/generator/form/sections/EducationSection.tsx:29]: Adding a preventDefault on the onclick seems to prevent entire Form re-renders.

**Comment by gabivlj** [comment]
LMK if this new version looks good! Happy to change anything you might see.

**Review by ducaale — APPROVED** [review]
Thanks!

**Pull request merged**